domains/authorization/Authorization.API/API/Authorization/Controllers/ConsentController.cs

```diff
@@ -15,8 +15,7 @@ namespace API.Authorization.Controllers;
 
 [ApiController]
 [ApiVersion(ApiVersions.Version20230101)]
-[Authorize(Policy.B2CCvrClaim)]
-[Authorize(Policy.B2CSubTypeUserPolicy)]
+[Authorize(Policy.Frontend)]
```

Comment:
Why arent 2rdParties allowed to access consent or terms?

Reply:
Only a user can grant consent and terms. Otherwise they would be able to give them self controll to grant consent to all organizations. So we need to know it's a user who has logged into MitID that grants a 3rd party consent or has accepted terms.

Reply:
Make sense :)